### PR TITLE
added webgpsmap plugin

### PR DESCRIPTION
**Pull request by xenDE**
## Description
a plugin for pwnagotchi that shows a openstreetmap with positions of ap-handshakes in the webbrowser

http://[ip-of-your-pwnagpotchi]:8080/plugins/webgpsmap

## How Has This Been Tested?
tested on master 
tested with net-pos plugin data by daPwnie
tested with gps plugin data by dadav

### CODE REVIEW

**Comment by Arttumiro** [comment]
Yoo this is amazing im gonna give it a test, looks really good

**Comment by xenDE** [comment]
fixed reported error if no position data exist in handshake directory

and needed to comment out the example gpio button commands in default.yml

File: pwnagotchi/defaults.yml [hunk 1]
```diff
@@ -30,6 +30,8 @@ main:
             enabled: false
             speed: 19200
             device: /dev/ttyUSB0
+        webgpsmap:
+            enabled: true
```
Comment (evilsocket) [inline pwnagotchi/defaults.yml]: Please set this to false by default.

**Comment by evilsocket** [comment]
By default it should be disabled

**Comment by xenDE** [comment]
sure, was a mistake.

**Comment by evilsocket** [comment]
thanks

**Comment by virtadpt** [comment]
Why not hook into gpsd?

**Pull request merged**